### PR TITLE
MS-668 Code style improvements

### PR DESCRIPTION
**Pull request by luhmirin-s**
## PR structure 
1. Commit with the KtLint and general formatting configuration 
2. Commit with fixes for issues that cannot be automatically fixed by the KtLint. The report can be generated by running 
`ktlint '**/*.kt' '!**/generated/**.kt' --format --reporter=html,output=build/ktlint-report.html`. The only unresolved issues are long lines. Fixes mostly fall into one of these categories:
    * Wildcard imports
    * Incorrect property name case
    * File contains a single class with a different name
    * Consecutive comments of different kinds
    * Empty file

3. Optional commit with automatically generated fixes (may not be present all of the time). I might not include this commit in some iterations to avoid polluting the UI, the steps below are the recommended way to do the review.

## How to review

Do not review this code in Github as the UI is not capable of handling 20k changes, instead do the following: 

1. Install KtLint CLI - https://pinterest.github.io/ktlint/latest/install/cli/ (do not install the plugin right now)
2. Check out the branch, but do a hard reset to the second commit
3. Run KtLint CLI command above
4. Review the changes in your favourite GIT diffing tool
5. If you find any objectable formatting - check if there is a rule that can disable this formatting - https://pinterest.github.io/ktlint/latest/rules/standard/ and propose a configuration update as a comment in the `.editorconfig` file (to allow discussion thread for specific proposal).

### CODE REVIEW

File: infra/event-sync/src/androidTest/java/com/simprints/infra/eventsync/remote/EventRemoteDataSourceImplAndroidTest.kt [hunk 1]
```diff
@@ -1,8 +1,10 @@
+// TODO fix
```
Comment (BurningAXE) [inline infra/event-sync/src/androidTest/java/com/simprints/infra/eventsync/remote/EventRemoteDataSourceImplAndroidTest.kt:1]: Will we ever? :D

File: infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt [hunk 2]
```diff
@@ -59,7 +56,7 @@ class SimApiClientImplTest {
 
     @Test
     fun `should throw a backend maintenance exception without estimated outage when it's a maintenance error without header and not retry`() =
-        runTest(StandardTestDispatcher()) {
+        runTest {
```
Comment (BurningAXE) [inline infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt:59]: Default is UnconfinedTestDispatcher!?

**Review by BurningAXE — APPROVED** [review]

File: infra/event-sync/src/androidTest/java/com/simprints/infra/eventsync/remote/EventRemoteDataSourceImplAndroidTest.kt (hunk 1 above)
Comment (luhmirin-s) [reply infra/event-sync/src/androidTest/java/com/simprints/infra/eventsync/remote/EventRemoteDataSourceImplAndroidTest.kt:1]: It is more likely do be deleted, but not in scope of this PR :D

File: feature/matcher/src/main/java/com/simprints/matcher/screen/MatchFragment.kt [hunk 3]
```diff
@@ -19,7 +19,7 @@ import com.simprints.infra.uibase.navigation.finishWithResult
 import com.simprints.infra.uibase.viewbinding.viewBinding
 import com.simprints.matcher.R
 import com.simprints.matcher.databinding.FragmentMatcherBinding
-import com.simprints.matcher.screen.MatchViewModel.MatchState.*
+import com.simprints.matcher.screen.MatchViewModel.MatchState
```
[the remark this answers is not in the document]
Comment (luhmirin-s) [reply feature/matcher/src/main/java/com/simprints/matcher/screen/MatchFragment.kt:22]: Looks clean to me, tbh. "MatchState" is not that long of a name to be unreadable and provides more context in params.

File: infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt (hunk 2 above)
Comment (luhmirin-s) [reply infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt:59]: From what I gathered in sources, it is `StandardTestDispatcher` by default, meaning this declaration was redundant.

Comment (BurningAXE) [reply infra/network/src/test/java/com/simprints/infra/network/apiclient/SimApiClientImplTest.kt:59]: Oh, my bad!

**Review by meladRaouf — APPROVED** [review]

**Review by alexandr-simprints — APPROVED** [review]

**Review by TristramN — APPROVED** [review]

**Review by alex-vt — APPROVED** [review]

**Pull request merged**